README.md

```diff
@@ -118,6 +118,15 @@ public void prepareConnection(@NotNull HttpURLConnection connection) {
 }
 ```
 
+### Can I use a proxy that will be used for uploading files?
+
+Yes, just add a proxy to the TusClient as shown in the above usage:
```

Comment:
above > below

Reply:
Reworded the text to link to the usage above and the example below.

Reply:
resovled